README.md

```diff
@@ -36,6 +36,33 @@
 
 Please note that this plugin is currently in alpha stage and still under active development. We encourage everyone to try it and give feedback, but we don't recommend it for production use yet.
 
+## Intro
```

Comment:
Maybe better heading? But i am bad for writing docs :smile:

Reply:
You're right, without any text the heading looked weird. I added a little introduction, mostly copied from Prettier core.